### ISSUE TITLE
Plot the Val loss and Training loss

### ISSUE THREAD

**Issue by Yasich** [Actis92/saint-lightning#9]
Hello, thank you so much for sharing this repo.
I want to plot validation and training loss. How can i do this in SAINT model? can you help me please?
@Actis92
